app/src/main/res/values/strings.xml

```diff
@@ -584,4 +584,7 @@
   <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
   <string name="revision_navigation_cards_header">Continue Studying</string>
   <string name="state_fragment_hint_bar_forced_announcement_text">Go to the bottom of the screen for a hint.</string>
+  <string name="please_select_all_correct_choices">Please select all correct choices.</string>
```

Comment:
The names need to make sense in the context of the entire app, and explain what the label is describing not the text of the label itself. For example, this one could be: ``state_fragment_item_selection_no_items_selected_hint_text``. Ditto for the other strings--please update them.

Reply:
okay